csFastFloat/FastFloatParser.cs

```diff
@@ -24,16 +24,13 @@ public static float exact_power_of_ten(long power)
 #endif
     }
 
-    public static unsafe float ToFloat(bool negative, AdjustedMantissa am)
+    public static float ToFloat(bool negative, AdjustedMantissa am)
     {
       ulong word = am.mantissa;
       word |= (ulong)(uint)(am.power2) << FloatBinaryConstants.mantissa_explicit_bits;
       word = negative ? word | ((ulong)(1) << FloatBinaryConstants.sign_index) : word;
-
-      float d = 0;
-      Unsafe.Copy(ref d, &word);
-
-      return d;
+      uint truncated_word = (uint) word;
+      return BitConverter.Int32BitsToSingle(truncated_word);
```

Comment:
```suggestion
      return BitConverter.Int32BitsToSingle((int)truncated_word);
```
To fix the build-error.

Or 
```c#
int truncated_word = (int)word;
return BitConverter.Int32BitsToSingle(truncated_word);
```